Model/Import/Attributes.php

```diff
@@ -110,7 +265,8 @@ public function parse($categoryFeaturesFile, $restrictedValuesFile, $productFeat
         $this->createAttributeGroups();
 
         //Parse features
-        $this->logger->info("Parsing category features file");
+        $this->output->writeln("-- Parsing category features file");
+        $this->logger->info("-- Parsing category features file");
```

Comment:
The dashes I used in "--- Begin Attribute Parse ---" served a purpose, which was to make an easy to spot delimiter in a log file (where it would be surrounded above and below by many lines of other content), putting dashes prior to every single output kinda defeats the object